adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/put-rum-metrics-destination.rst

> **To creates or updates a destination to receive extended metrics from CloudWatch RUM**

Reviewer: ```suggestion
**To create or update a destination to receive extended metrics from CloudWatch RUM**
```

> The following ``put-rum-metrics-destination`` creates or updates a destination to receive extended metrics from CloudWatch RUM. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``put-rum-metrics-destination`` example creates or updates a destination to receive extended metrics from CloudWatch RUM. ::
```

> 	aws rum put-rum-metrics-destination \

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--app-monitor-name MyWebApp \
		--destination Evidently \
		--destination-arn arn:aws:evidently:us-east-1:123456789012:project/petfood/feature/petfood-upsell-text --iam-role-arn arn:aws:iam::123456789012:role/Assume-role-EC2 

> 

Reviewer: ```suggestion

This command produces no output.

```
